src/client/lib/render-vue.js

```diff
@@ -0,0 +1,8 @@
+import Vue from 'vue'
```

Comment:
would be nice to have some docs in this file to explain why it exists

Reply:
will add it